h/templates/5xx.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html>
+  <head>
+      <meta charset="UTF-8" />
+      <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1"/>
+      <meta name="viewport" content="width=device-width,initial-scale=1" />
+      <title>Server Error</title>
+      <style>
```

Comment:
The plan originally was to make it completely standalone, now it depends on the server for images so this can also go.